caddyconfig/httpcaddyfile/tlsapp.go

```diff
@@ -533,7 +533,7 @@ func consolidateAutomationPolicies(aps []*caddytls.AutomationPolicy) []*caddytls
 						}
 					}
 					aps = append(aps[:j], aps[j+1:]...)
-					j--
+					j = max(0, j-1)
```

Comment:
I don't think `j` will ever be 0, right?

Reply:
You're right, cause it's always `j=i+1`.